op-e2e/e2eutils/geth/wait.go

```diff
@@ -86,6 +87,31 @@ func WaitForTransaction(hash common.Hash, client *ethclient.Client, timeout time
 	}
 }
 
+// WaitForBlockWithTxFromSender waits for a block with a transaction from a specific sender address.
+// It starts from the current block and checks the next nBlocks blocks.
```

Comment:
Nit: this doc doesn't mention what's returned

Reply:
[9d9d7b9](https://github.com/Layr-Labs/optimism/pull/34/commits/9d9d7b90532ca02220251852b2cc78404d7565f9)